### ISSUE TITLE
Feature request: reduce outputs

### ISSUE THREAD

**Issue by cyanglaz** [lewagon/wait-on-check-action#31]
The wait-on-check produces a lot of outputs. Sometimes it made the GitHub web UI hang when trying to see the result. It would be nice to have a flag to make the output less verbose.
Maybe something like:
```
    - name: Wait on all tests
      uses: lewagon/wait-on-check-action@master
      with:
        ref: ${{ github.sha }}
        running-workflow-name: 'xxx'
        repo-token: ${{ secrets.GITHUB_TOKEN }}
        wait-interval: 180 # seconds
        allowed-conclusions: success
        verbose: false
```

Maybe the output can just be like:
```
Waited for 312 seconds.
```

**Comment by progapandist** [comment]
Thank you for reporting. I will find time to take a look at it, but for now I'm a bit swamped with work projects, so I wouldn't expect a quick resolution, thank you for understanding!

**Comment by cyanglaz** [comment]
@progapandist Thanks for the quick response. 
Do you think `verbose: false` (true by default) is a good solution to this? 
Or maybe `output: false` (true by default), which omits all the outputs.

**Comment by progapandist** [comment]
I'd go with verbose!

**Comment by ditman** [comment]
I've had more luck by looking at the Raw logs, instead of attempting to read them through the Github interface:

<img width="919" alt="Screen Shot 2021-08-12 at 11 25 55 AM" src="https://user-images.githubusercontent.com/1255594/129253023-482fd3ec-1a5f-4647-ba1f-99ab73eb4ee1.png">

_(Important bit: do not attempt to open the "Wait on all tests" step, that's the part that breaks the browser!)_

**Comment by ditman** [comment]
Also: conditionally not printing the full output of the Checks API call [here](https://github.com/lewagon/wait-on-check-action/blob/3ac63078fe8644f15c807e6419c8a8a1ec7913f1/app/services/github_checks_verifier.rb#L25) would go a looooong way to fixing the issue, IMO.

Or, to not lose much usability of the non-verbose logs, printing an abridged version of the API response, with the name of the checks that are still running, or similar would be nice at this point. Something like this:

`5 of 28 checks completed. Still waiting for: check 2, check 5, check-whatever-else, ..., last-but-not-least-check-28.`

**Comment by matiasalbarello** [comment]
Fixed on #42
